src/ontodia/data/rdf/rdfDataProvider.ts

```diff
@@ -562,28 +557,13 @@ export class RDFDataProvider implements DataProvider {
         });
     };
 
-    private createLabelFromId(id: string): LocalizedString[] {
-        let label;
-        if (id) {
-            const urlParts = id.split('/');
-            const sharpParts = urlParts[urlParts.length - 1].split('#');
-            label = sharpParts[sharpParts.length - 1];
-        } else {
-            label = '';
-        }
-        return [{
-                text: label,
-                lang: '',
-            }];
-    }
-
     private getLabels(id: string): Promise<LocalizedString[]> {
         return this.rdfStorage.getLabels(id).then(labelTriples => {
             const tripleArray = labelTriples.toArray();
             return tripleArray.length > 0 ? labelTriples.toArray().map(l => ({
                 text: l.object.nominalValue,
                 lang: isLiteral(l.object) ? l.object.language || '' : '',
-            })) : this.createLabelFromId(id);
+            })) : [];
```

Comment:
You can safely omit conditional operator here and replace this statement with just
```js
return tripleArray.map(l => ({
    text: l.object.nominalValue,
    lang: isLiteral(l.object) ? l.object.language || '' : '',
}));
```